crates/notion-core/src/tool/mod.rs

```diff
@@ -27,10 +29,11 @@ use self::npm::Npm;
 use self::npx::Npx;
 use self::yarn::Yarn;
 
+#[derive(PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
Including all of these up to `Ord` means the sequence defined here will be used for sorting when we have a `Vec<ToolSpec>`. I note that this is not obvious, so will add a comment explaining as much here.